crates/iroha/src/lib.rs

```diff
@@ -9,4 +9,4 @@ mod secrecy;
 
 pub use iroha_crypto as crypto;
 pub use iroha_data_model as data_model;
-pub use iroha_multisig_data_model as multisig_data_model;
+pub use iroha_executor_data_model as executor_data_model;
```

Comment:
Now `iroha` re-exports `::executor_data_model` so that clients can reference the multisig data model. Is it okay @mversic  ?

Reply:
I think it's ok